lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -82,10 +82,12 @@ defmodule ExUnit.Filters do
   """
   @spec eval(t, t, map, [ExUnit.Test.t]) :: :ok | {:error, binary}
   def eval(include, exclude, tags, collection) when is_map(tags) do
+    include_skip? = Enum.any?(include, &has_tag(&1, %{skip: true}, collection))
```

Comment:
Would this look better reversed in order to avoid the negations below with something like:

```
    skip? = Enum.all?(include, &has_tag(&1, %{skip: true}, collection) == false)
```